{"uid":"f9398cd3d450f58f","name":"test_exporter_endpoint_with_tls","fullName":"tests.integration.test_exporter#test_exporter_endpoint_with_tls","historyId":"f606b20e23f9624d312739f189214496","time":{"start":1720750413671,"stop":1720750413671,"duration":0},"description":"Test that the exporter endpoint works when related with TLS","descriptionHtml":"<p>Test that the exporter endpoint works when related with TLS</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 157, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 497, in setup\n    raise exc\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 494, in setup\n    col.setup()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1791, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 566, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 585, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 607, in _get_active_fixturedef\n    self._compute_fixture_value(fixturedef)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 693, in _compute_fixture_value\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1069, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 216, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 256, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1720750122676,"stop":1720750122676,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1720750122677,"stop":1720750122677,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1720750122677,"stop":1720750122952,"duration":275},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1720750413672,"stop":1720750413672,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 216, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 256, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Test that the exporter endpoint works when related with TLS","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 157, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 497, in setup\n    raise exc\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 494, in setup\n    col.setup()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1791, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 566, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 585, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 607, in _get_active_fixturedef\n    self._compute_fixture_value(fixturedef)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 693, in _compute_fixture_value\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1069, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 216, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 256, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","steps":[],"attachments":[{"uid":"1acc9226cc4220db","name":"log","source":"1acc9226cc4220db.txt","type":"text/plain","size":6698}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"ops_test::finalizer","time":{"start":1720750413674,"stop":1720750433690,"duration":20016},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1720750433691,"stop":1720750433691,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1720750433691,"stop":1720750433691,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1720750433691,"stop":1720750433691,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_exporter"},{"name":"host","value":"fv-az1542-939"},{"name":"thread","value":"12403-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_exporter"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"5a507ad529d2caea","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":10,"broken":0,"skipped":9,"passed":21,"unknown":4,"total":44},"items":[{"uid":"11b69a3936425f1d","reportUrl":"../151//#testresult/11b69a3936425f1d","status":"passed","time":{"start":1720663994780,"stop":1720664056780,"duration":62000}},{"uid":"6b44c07aa558936f","reportUrl":"../148//#testresult/6b44c07aa558936f","status":"failed","statusDetails":"AssertionError: Expected mysqlrouter certificate from self-signed-certificates\nassert 'CN = Test CA' in '        Issuer: CN = MySQL_Router_Auto_Generated_CA_Certificate\\n'","time":{"start":1720577617880,"stop":1720577665729,"duration":47849}},{"uid":"2cbf4a491c6cdb6d","reportUrl":"../147//#testresult/2cbf4a491c6cdb6d","status":"passed","time":{"start":1720491156967,"stop":1720491224305,"duration":67338}},{"uid":"f764b40eee4b2485","reportUrl":"../145//#testresult/f764b40eee4b2485","status":"failed","statusDetails":"AssertionError: Expected mysqlrouter certificate from self-signed-certificates\nassert 'CN = Test CA' in '        Issuer: CN = MySQL_Router_Auto_Generated_CA_Certificate\\n'","time":{"start":1720404830983,"stop":1720404876866,"duration":45883}},{"uid":"18318ffccf8a9a49","reportUrl":"../144//#testresult/18318ffccf8a9a49","status":"passed","time":{"start":1720318631456,"stop":1720318698023,"duration":66567}},{"uid":"afb458feccb66290","reportUrl":"../143//#testresult/afb458feccb66290","status":"passed","time":{"start":1720231675843,"stop":1720231737110,"duration":61267}},{"uid":"5f0bdf44c12aa856","reportUrl":"../142//#testresult/5f0bdf44c12aa856","status":"passed","time":{"start":1720145437183,"stop":1720145505006,"duration":67823}},{"uid":"5297374c8e12e14","reportUrl":"../140//#testresult/5297374c8e12e14","status":"failed","statusDetails":"AssertionError: Expected mysqlrouter certificate from self-signed-certificates\nassert 'CN = Test CA' in '        Issuer: CN = MySQL_Router_Auto_Generated_CA_Certificate\\n'","time":{"start":1720059068708,"stop":1720059111229,"duration":42521}},{"uid":"ade5454bee5e850c","reportUrl":"../139//#testresult/ade5454bee5e850c","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1719972797815,"stop":1719972797815,"duration":0}},{"uid":"4bc609c9c4b652f1","reportUrl":"../137//#testresult/4bc609c9c4b652f1","status":"failed","statusDetails":"AssertionError: Expected mysqlrouter certificate from self-signed-certificates\nassert 'CN = Test CA' in '        Issuer: CN = MySQL_Router_Auto_Generated_CA_Certificate\\n'","time":{"start":1719886296873,"stop":1719886343620,"duration":46747}},{"uid":"6f0b0e3781368f6e","reportUrl":"../135//#testresult/6f0b0e3781368f6e","status":"passed","time":{"start":1719800339113,"stop":1719800407679,"duration":68566}},{"uid":"ca8495013bc054a1","reportUrl":"../134//#testresult/ca8495013bc054a1","status":"passed","time":{"start":1719713766281,"stop":1719713833682,"duration":67401}},{"uid":"463373f35c48f520","reportUrl":"../133//#testresult/463373f35c48f520","status":"failed","statusDetails":"AssertionError: Expected mysqlrouter certificate from self-signed-certificates\nassert 'CN = Test CA' in '        Issuer: CN = MySQL_Router_Auto_Generated_CA_Certificate\\n'","time":{"start":1719626926007,"stop":1719626975559,"duration":49552}},{"uid":"8312f727bdbe5d32","reportUrl":"../131//#testresult/8312f727bdbe5d32","status":"failed","statusDetails":"AssertionError: Expected mysqlrouter certificate from self-signed-certificates\nassert 'CN = Test CA' in '        Issuer: CN = MySQL_Router_Auto_Generated_CA_Certificate\\n'","time":{"start":1719540849569,"stop":1719540896627,"duration":47058}},{"uid":"85c1fe5c8b5cf9c3","reportUrl":"../130//#testresult/85c1fe5c8b5cf9c3","status":"failed","statusDetails":"AssertionError: Expected mysqlrouter certificate from self-signed-certificates\nassert 'CN = Test CA' in '        Issuer: CN = MySQL_Router_Auto_Generated_CA_Certificate\\n'","time":{"start":1719454237497,"stop":1719454285247,"duration":47750}},{"uid":"92bd7274a306c3b1","reportUrl":"../128//#testresult/92bd7274a306c3b1","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1719367906897,"stop":1719367906897,"duration":0}},{"uid":"f00aabd8152bc2d4","reportUrl":"../126//#testresult/f00aabd8152bc2d4","status":"failed","statusDetails":"AssertionError: Expected mysqlrouter certificate from self-signed-certificates\nassert 'CN = Test CA' in '        Issuer: CN = MySQL_Router_Auto_Generated_CA_Certificate\\n'","time":{"start":1719281565850,"stop":1719281614600,"duration":48750}},{"uid":"65d21e5530cd5fbd","reportUrl":"../123//#testresult/65d21e5530cd5fbd","status":"failed","statusDetails":"AssertionError: Expected mysqlrouter certificate from self-signed-certificates\nassert 'CN = Test CA' in '        Issuer: CN = MySQL_Router_Auto_Generated_CA_Certificate\\n'","time":{"start":1719195105448,"stop":1719195143381,"duration":37933}},{"uid":"688ca39fb33ee7d9","reportUrl":"../122//#testresult/688ca39fb33ee7d9","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1719108977669,"stop":1719108977669,"duration":0}},{"uid":"11c984e7f5feef8d","reportUrl":"../121//#testresult/11c984e7f5feef8d","status":"passed","time":{"start":1719022025711,"stop":1719022093653,"duration":67942}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"f9398cd3d450f58f.json","parameterValues":[]}